Makefile

```diff
@@ -147,8 +147,7 @@ cluster-deploy: cluster-clean
 	./hack/cluster-deploy.sh
 
 cluster-sync:
-	$(MAKE) cluster-build
```

Comment:
Commit msg: you said: "run the cluster-sync parallel to the cluster-build",  but the current cluster-sync runs cluster-build.
I think you meant  "run cluster-clean parallel to the cluster-build"?

Reply:
right will fix, thanks